Tests/BasicPerformanceTests/SHA256PerfTests.swift

```diff
@@ -0,0 +1,31 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import XCTest
+
+import Basic
+import TestSupport
+
+class SHA256PerfTests: XCTestCasePerf
+{
+    func test() {
```

Comment:
A little descriptive name will be nice. something like `test20MBDigest_X1000` in this case.